tests/integration/device_test.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
```

Comment:
I think some dependency is missing for this test because when I run it, I see some log messages:

![Screenshot from 2023-12-16 10-47-58](https://github.com/kworkflow/kworkflow/assets/3153524/fa012b1c-d267-499f-89a5-4482d4907904)

Reply:
Yes, but I do not know why this is not working in your machine and is working on my machine because we install all kw's dependencies while building the container images...

I'll try to reproduce this behavior in a Debian VM. If the issue is due to missing dependencies in the dependency lists, I guess the proper action is to open a new issue and solve it there.

Reply:
I can reproduce it locally.

Reply:
PR #1001 fixes this